static/dashboard.html

```diff
@@ -49,6 +55,11 @@ <h3>Table reservation chart</h3>
 
     function createRemindButton(bookingId) {
       var button = document.createElement("button");
+      if (bookingId == undefined){
+        button.className = "btn btn-primary";
+      } else {
+        button.className = "btn btn-success";
```

Comment:
Can you color buttons based on booking id? Let's say having list with colors and booking id will be the index of it? That may improve visibility of which tables are booked together

Reply:
Good idea, but are we supporting multiple tables for a single customer?

Reply:
Ah I see what you mean.

Reply:
If we implement PAX, then this might be worth doing, but I'm not sure it's worth it for now.

Reply:
Yes, this is why I asked, because didn't want to mess with bootstrap now ^^